src/server/sim-files.js

```diff
@@ -139,7 +139,12 @@ SimulationFiles.prototype._createHostJsFile = function (simulationFilePath, host
 
     var scriptDefs = createScriptDefs(hostType, scriptTypes);
 
-    var b = browserify({ paths: getBrowserifySearchPaths(hostType), debug: true });
+    var b = browserify({
+        paths: getBrowserifySearchPaths(hostType),
+        debug: true,
+        externalRequireName: '_cordovaSimulateRequire'
```

Comment:
I passed that option, `hasExports` to false, instead of `externalRequireName`, and nothing happen, `require` is still exports to the global. I will research this.